### PR TITLE
Upload August 2018 QAT Gazette

### PR DESCRIPTION
**Pull request by JBHyperion**
---

### CODE REVIEW

File: news/2018-09-18-the-qat-gazette-august-2018.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
+---
+layout: post
+title: "The QAT Gazette: August 2018"
+date: 2018-08-18 12:00:00 +0000
+---
+
+Modding discussion feature requests, potential changes to hybrid mode mapsets, and an update on awaited Beatmap Nominator applications — all your mapping and modding news from the past month is inside!
```
[the remark this answers is not in the document]
Comment (JBHyperion) [reply news/2018-09-18-the-qat-gazette-august-2018.md]: Today I learned

File: news/2018-09-18-the-qat-gazette-august-2018.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+
+![](/wiki/shared/news/banners/qat-blog-logo.png)
+
+## Opening
+by [JBHyperion](https://osu.ppy.sh/users/4879508)
+
+Hello again!
+
+The summer vacation is coming to an end; for most of us heralding the return to school, university or work. Whether this is inspires excitement, dread, or any emotion in-between, a healthy dose of mapping and modding news updates is surely just what you need! Whether you're here to keep up to date with Beatmap Nominator rule changes, new and upcoming Ranking Criteria updates, or recent meetings of the Quality Assurance Team, we've got you covered. Scroll down to find your own little slice of information heaven!
```
[the remark this answers is not in the document]
Comment (JBHyperion) [reply news/2018-09-18-the-qat-gazette-august-2018.md]: yes

[the remark this answers is not in the document]
Comment (JBHyperion) [reply news/2018-09-18-the-qat-gazette-august-2018.md]: Typos are fun

[the remark this answers is not in the document]
Comment (JBHyperion) [reply news/2018-09-18-the-qat-gazette-august-2018.md]: Agreed, just removed the second instance

File: news/2018-09-18-the-qat-gazette-august-2018.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+
+Before we begin though, please join me in showing the utmost appreciation for some of our longest serving contributors in the Beatmap Nominators — [alienflybot](https://osu.ppy.sh/users/636114), [Kagetsu](https://osu.ppy.sh/users/6203841), [sahuang](https://osu.ppy.sh/users/5318910) and [Strategas](https://osu.ppy.sh/users/2971837) (2 years of service each), [Kibbleru](https://osu.ppy.sh/users/3193504) (3 years) and [Gero](https://osu.ppy.sh/users/1467715), [jonathanlfj](https://osu.ppy.sh/users/270377), [Natsu](https://osu.ppy.sh/users/1953876) and [smallboat](https://osu.ppy.sh/users/243049), each with a staggering **four years** of cumulative service.
```
[the remark this answers is not in the document]
Comment (JBHyperion) [reply news/2018-09-18-the-qat-gazette-august-2018.md]: Consistency

File: news/2018-09-18-the-qat-gazette-august-2018.md [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+
+From the Quality Assurance Team, we would also like to recognize the incomparable efforts of [Deif](https://osu.ppy.sh/users/318565) and [p3n](https://osu.ppy.sh/users/123703), who have also served admirably in their respective field(s) for an incredible **four years**.
```
[the remark this answers is not in the document]
Comment (JBHyperion) [reply news/2018-09-18-the-qat-gazette-august-2018.md]: Didn't realise I used also twice here, fixed

File: news/2018-09-18-the-qat-gazette-august-2018.md [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+
+This gives future applicants an opportunity to increase their modding score in the meantime, to ensure they meet the threshold once applications do come out. Regardless of when applications re-open, consistent effort and contribution will always be highly valued, so make sure to continue modding regularly in the meantime if you have aspirations to join the team!
+
+### Beatmap Nominator Retirements
+
+This month saw the departure of six of our hard working Beatmap Nominators; some old, some relatively new — [Ascendance](https://osu.ppy.sh/users/2931883), [Cerulean Veyron](https://osu.ppy.sh/users/1886524), [Fii](https://osu.ppy.sh/users/3922569),  [Garalulu](https://osu.ppy.sh/users/757783), [riffy](https://osu.ppy.sh/users/597957) and [Sieg](https://osu.ppy.sh/users/1404615). Whether it’s for university study, military service, or simply a well needed rest, your contributions have been greatly appreciated and we wish you the best of luck in whichever path you now pursue.
```
[the remark this answers is not in the document]
Comment (JBHyperion) [reply news/2018-09-18-the-qat-gazette-august-2018.md]: added hyphens

[the remark this answers is not in the document]
Comment (JBHyperion) [reply news/2018-09-18-the-qat-gazette-august-2018.md]: Alright

File: news/2018-09-18-the-qat-gazette-august-2018.md [hunk 6]
```diff
@@ -0,0 +1,74 @@
...
+
+## Within the Quality Assurance Team
+by [Raiden](https://osu.ppy.sh/users/2239480)
+
+Most of this last month's news and developments centred the discussion and outcomes of a Quality Assurance Team Meeting. Usually we like to hold these publically, but sometimes time isn't on our side and we really need to discuss things at short notice, so apologies for that! A recap of the key aspects of this meeting can be found below.
```
[the remark this answers is not in the document]
Comment (JBHyperion) [reply news/2018-09-18-the-qat-gazette-august-2018.md]: De-capitalised

[the remark this answers is not in the document]
Comment (JBHyperion) [reply news/2018-09-18-the-qat-gazette-august-2018.md]: Interesting, I think I might have been spelling this wrong for about 20 years lol

[the remark this answers is not in the document]
Comment (cl8n) [reply news/2018-09-18-the-qat-gazette-august-2018.md]: yep, it's one of those gotcha's in english. I usually get this wrong too if a red underline doesn't remind me

File: news/2018-09-18-the-qat-gazette-august-2018.md [hunk 7]
```diff
@@ -0,0 +1,74 @@
...
+layout: post
+title: "The QAT Gazette: August 2018"
+date: 2018-09-18 10:00:00 +0000
+---
+
+Modding discussion feature requests, potential changes to hybrid mode mapsets, and an update on awaited Beatmap Nominator applications—all your mapping and modding news from the past month is inside!
+
+![](/wiki/shared/news/banners/qat-blog-logo.png)
+
+## Opening
```
Comment (TPGPL) [inline news/2018-09-18-the-qat-gazette-august-2018.md]: Empty line after header

File: news/2018-09-18-the-qat-gazette-august-2018.md [hunk 8]
```diff
@@ -0,0 +1,74 @@
...
+
+From the Quality Assurance Team, we would also like to recognize the incomparable efforts of [Deif](https://osu.ppy.sh/users/318565) and [p3n](https://osu.ppy.sh/users/123703), who have served admirably in their respective fields for an incredible **four years**.
+
+It is no exaggeration to state that their tireless efforts have helped shape osu!'s mapping and modding communities into the successful force they are today, having assisted and mentored a countless number of mappers along the way. Each will receive a profile badge advertising their length of service which they are encouraged to display with pride!
+
+## Within the Beatmap Nominators
```
Comment (TPGPL) [inline news/2018-09-18-the-qat-gazette-august-2018.md]: Empty line after header

File: news/2018-09-18-the-qat-gazette-august-2018.md [hunk 9]
```diff
@@ -0,0 +1,74 @@
...
+
+This gives future applicants an opportunity to increase their modding score in the meantime, to ensure they meet the threshold once applications do come out. Regardless of when applications re-open, consistent effort and contribution will always be highly valued, so make sure to continue modding regularly in the meantime if you have aspirations to join the team!
+
+### Beatmap Nominator Retirements
+
+This month saw the departure of six of our hard-working Beatmap Nominators: some old, some relatively new—[Ascendance](https://osu.ppy.sh/users/2931883), [Cerulean Veyron](https://osu.ppy.sh/users/1886524), [Fii](https://osu.ppy.sh/users/3922569),  [Garalulu](https://osu.ppy.sh/users/757783), [riffy](https://osu.ppy.sh/users/597957) and [Sieg](https://osu.ppy.sh/users/1404615). Whether it’s for university study, military service, or simply a well-needed rest, your contributions have been greatly appreciated and we wish you the best of luck in whichever path you now pursue.
+
+## Within the Quality Assurance Team
```
Comment (TPGPL) [inline news/2018-09-18-the-qat-gazette-august-2018.md]: Empty line after header

File: news/2018-09-18-the-qat-gazette-august-2018.md [hunk 10]
```diff
@@ -0,0 +1,74 @@
...
+
+Hybrid mapsets have recently been seen as a “shortcut” to the Ranked section, reducing the robustness of the beatmap checking process by requiring only one Nominator per game mode. They are also slowly becoming obsolete, considering that all game modes now have more than sufficient levels of content in the Ranked section.
+
+These aspects caused us to reconsider if we really need to continue accommodating hybrid mapsets, and as such, we will be holding further discussion and possibly another community poll on this topic in the near future in order to decide how to proceed.
+
+### New Quality Assurance Team additions
+
+Please give a warm welcome to our newest osu!catch roster addition: [ZiRoX](https://osu.ppy.sh/users/200768)! A long-serving Beatmap Nominator of 3 years, who had a primary role in the remaking of the osu!catch Ranking Criteria, has joined us in order to assist with disqualifications, future Beatmap Nominator applications and evaluations. Make sure to wish him luck as he settles into his new role!
+
+## Summary
```
Comment (TPGPL) [inline news/2018-09-18-the-qat-gazette-august-2018.md]: Empty line after header

File: news/2018-09-18-the-qat-gazette-august-2018.md [hunk 11]
```diff
@@ -0,0 +1,74 @@
...
+
+### Beatmap Nominator Retirements
+
+This month saw the departure of six of our hard-working Beatmap Nominators: some old, some relatively new—[Ascendance](https://osu.ppy.sh/users/2931883), [Cerulean Veyron](https://osu.ppy.sh/users/1886524), [Fii](https://osu.ppy.sh/users/3922569),  [Garalulu](https://osu.ppy.sh/users/757783), [riffy](https://osu.ppy.sh/users/597957) and [Sieg](https://osu.ppy.sh/users/1404615). Whether it’s for university study, military service, or simply a well-needed rest, your contributions have been greatly appreciated and we wish you the best of luck in whichever path you now pursue.
+
+## Within the Quality Assurance Team
+by [Raiden](https://osu.ppy.sh/users/2239480)
+
+Most of this last month's news and developments centred the discussion and outcomes of a Quality Assurance Team meeting. Usually we like to hold these publicly, but sometimes time isn't on our side and we really need to discuss things at short notice, so apologies for that! A recap of the key aspects of this meeting can be found below.
```
Comment (Raidencio) [inline news/2018-09-18-the-qat-gazette-august-2018.md]: centred -> centered (not that it is wrong it's just that I'd never say centred so it doesn't look like I wrote this)

File: news/2018-09-18-the-qat-gazette-august-2018.md [hunk 12]
```diff
@@ -0,0 +1,74 @@
+---
+layout: post
+title: "The QAT Gazette: August 2018"
+date: 2018-09-18 10:00:00 +0000
+---
+
+Modding discussion feature requests, potential changes to hybrid mode mapsets, and an update on awaited Beatmap Nominator applications—all your mapping and modding news from the past month is inside!
```
Comment (TPGPL) [inline news/2018-09-18-the-qat-gazette-august-2018.md]: Space before and after hyphen

File: news/2018-09-18-the-qat-gazette-august-2018.md [hunk 13]
```diff
@@ -0,0 +1,74 @@
...
+
+This gives future applicants an opportunity to increase their modding score in the meantime, to ensure they meet the threshold once applications do come out. Regardless of when applications re-open, consistent effort and contribution will always be highly valued, so make sure to continue modding regularly in the meantime if you have aspirations to join the team!
+
+### Beatmap Nominator Retirements
+
+This month saw the departure of six of our hard-working Beatmap Nominators: some old, some relatively new—[Ascendance](https://osu.ppy.sh/users/2931883), [Cerulean Veyron](https://osu.ppy.sh/users/1886524), [Fii](https://osu.ppy.sh/users/3922569),  [Garalulu](https://osu.ppy.sh/users/757783), [riffy](https://osu.ppy.sh/users/597957) and [Sieg](https://osu.ppy.sh/users/1404615). Whether it’s for university study, military service, or simply a well-needed rest, your contributions have been greatly appreciated and we wish you the best of luck in whichever path you now pursue.
```
Comment (TPGPL) [inline news/2018-09-18-the-qat-gazette-august-2018.md]: Space before and after hyphen

File: news/2018-09-18-the-qat-gazette-august-2018.md [hunk 14]
```diff
@@ -0,0 +1,74 @@
...
+
+Hello again!
+
+The summer vacation is coming to an end, for most of us heralding the return to school, university or work. Whether this inspires excitement, dread, or any emotion in-between, a healthy dose of mapping and modding news updates is surely just what you need! If you're here to keep up to date with Beatmap Nominator rule changes, new and upcoming Ranking Criteria updates, or recent meetings of the Quality Assurance Team, we've got you covered. Scroll down to find your own little slice of information heaven!
+
+Before we begin though, please join me in showing the utmost appreciation for some of our longest serving contributors in the Beatmap Nominators—[alienflybot](https://osu.ppy.sh/users/636114), [Kagetsu](https://osu.ppy.sh/users/6203841), [sahuang](https://osu.ppy.sh/users/5318910) and [Strategas](https://osu.ppy.sh/users/2971837) (two years of service each), [Kibbleru](https://osu.ppy.sh/users/3193504) (three years) and [Gero](https://osu.ppy.sh/users/1467715), [jonathanlfj](https://osu.ppy.sh/users/270377), [Natsu](https://osu.ppy.sh/users/1953876) and [smallboat](https://osu.ppy.sh/users/243049), each with a staggering **four years** of cumulative service.
```
Comment (TPGPL) [inline news/2018-09-18-the-qat-gazette-august-2018.md]: Space before and after hyphen

File: news/2018-09-18-the-qat-gazette-august-2018.md [hunk 15]
```diff
@@ -0,0 +1,74 @@
...
+## Within the Quality Assurance Team
+by [Raiden](https://osu.ppy.sh/users/2239480)
+
+Most of this last month's news and developments centred the discussion and outcomes of a Quality Assurance Team meeting. Usually we like to hold these publicly, but sometimes time isn't on our side and we really need to discuss things at short notice, so apologies for that! A recap of the key aspects of this meeting can be found below.
+
+### QAT September flash-meeting
+
+The flash-meeting was held in order to follow up on the somewhat unfinished topics of our [public August meeting](https://osu.ppy.sh/community/forums/topics/792022), covering both these and new issues that required our attention.
+
+The most notable discussed topic was the making of a *feature request listing*, directed at the osu! developers. We brainstormed many quality-of-life implementations that we believe would make our job as QATs a smoother and more productive experience, some of which are listed below:
```
Comment (TPGPL) [inline news/2018-09-18-the-qat-gazette-august-2018.md]: feature requests

File: news/2018-09-18-the-qat-gazette-august-2018.md [hunk 16]
```diff
@@ -0,0 +1,69 @@
...
+
+From the Quality Assurance Team, we would also like to recognize the incomparable efforts of [Deif](https://osu.ppy.sh/users/318565) and [p3n](https://osu.ppy.sh/users/123703), who have served admirably in their respective fields for an incredible **four years**.
+
+It is no exaggeration to state that their tireless efforts have helped shape osu!'s mapping and modding communities into the successful force they are today, having assisted and mentored a countless number of mappers along the way. Each will receive a profile badge advertising their length of service which they are encouraged to display with pride!
```
Comment (osuShiro) [inline news/2018-09-18-the-qat-gazette-august-2018.md]: I'm not sure "advertising" is the right word here. Think more "recognising" or "acknowledging"

Comment (JBHyperion) [reply news/2018-09-18-the-qat-gazette-august-2018.md]: Acknowledging is good, changed

**Pull request merged**